tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -12,7 +12,11 @@ CREATE TABLE IF NOT EXISTS universe_roots (
 
     -- We use the 32 byte schnorr key here as this is what's used to derive the
     -- top-level Taproot Asset commitment key.
-    group_key BLOB CHECK(LENGTH(group_key) = 32)
+    group_key BLOB CHECK(LENGTH(group_key) = 32),
+
+    -- This field is an enum representing the proof type stored in the given
+    -- universe.
+    proof_type TEXT NOT NULL CHECK(proof_type IN ('issuance', 'transfer'))
```

Comment:
Also IIUC, `namespace_root` now also includes this as a prefix. May want to update some of the comments in the schema file.

Reply:
Right now, the schema doesn't explain what's in `namespace_root`:
```
    -- For the namespace root, we set the foreign key constraint evaluation to
    -- be deferred until after the database transaction ends. Otherwise, if the
    -- root of the SMT is deleted temporarily before inserting a new root, then
    -- this constraint is violated as there's no longer a root that this
    -- universe tree can point to.
    namespace_root VARCHAR UNIQUE NOT NULL REFERENCES mssmt_roots(namespace) DEFERRABLE INITIALLY DEFERRED,
 ```